src/common/LogEntry.h

```diff
@@ -173,8 +171,8 @@ inline ostream& operator<<(ostream& out, const clog_type t)
 
 inline ostream& operator<<(ostream& out, const LogEntry& e)
 {
-  return out << e.stamp << " " << e.name << " " << e.who
-	     << " " << e.seq << " : "
+  return out << e.stamp << " " << e.name << " (" << e.rank
+	     << " " << e.addrs << ") " << e.seq << " : "
```

Comment:
do we really want to print out the `addrs` in each log message? i think this slows down the logging even more.